gtk/src/widgets/dialogs/mod.rs

```diff
@@ -23,7 +23,7 @@ fn dialog_template<F: FnOnce(&gtk::Box)>(
         .accept_focus(true)
         .deletable(true)
         .destroy_with_parent(true)
-        .use_header_bar(1i32)
+        .use_header_bar(1_i32)
```

Comment:
This is a pedantic lint, which isn't enabled and so not required in this change.

Reply:
actually there's a few refactors in this pull request that aren't strictly required to satisfy the `clippy::all` lint. is that an issue?

Reply:
I think if they're being changed then they should have the corresponding lint enabled. The whole set may be a lot of noise, so it may be better to pick a subset to enforce.

Reply:
done.

Reply:
The rule was added without fixing it; clippy will now fail. Only 3 instances were changed, so it's easier to just drop this lint.